agent/app/args/flag.go

```diff
@@ -13,11 +13,14 @@
 
 package args
 
-import "flag"
+import (
+	"flag"
+)
 
 const (
 	versionUsage             = "Print the agent version information and exit"
-	logLevelUsage            = "Loglevel: [<crit>|<error>|<warn>|<info>|<debug>]"
```

Comment:
Who provides these flags? I'm wondering if removing this flag can potentially impact customers if they have some custom setting where they interact directly with agent.

Reply:
These are the flags the user can provide for running agent with the docker exec command. Do you think it would be better to have the old flag and also the 2 new ones, and if the old one is set, use that instead?

Reply:
sorry I meant to tag this comment where "logLevelFlagName" variable is at line# 29 (new line# 32) where the flag name changed from "loglevel" to "loglevel-driver", but I think you got what I meant. 
if this is user facing, removing a flag is a backward breaking change. I think what you suggested should work.